### ISSUE TITLE
User is able to set current password again

### ISSUE THREAD

**Issue by chucki79** [unosquare/passcore#212]
I know that you are using the same procedure like an account operator would do changing the user password within ADUC console. 
The problem is, that Users are able to set their old/current password as their new password. So the password history is not really checked. Is there any possibility to integrate this?

**Comment by geoperez** [comment]
Do you have your AD with the policy to `Enforce password history`? 

PassCore doesn't keep track of any password, if Active Directory doesn't enforce password history, we can not help you.

**Comment by chucki79** [comment]
Yes we do have enforce password history active. Also minimum password age 1 day etc. I am using this tool to change passwords for remote domains. 
In appsettings.json we use a User that does have permissions in that remote domain to change/reset passwords. So it seems that passcore always makes a password reset instead of a change. Using a reset obviosly makes it possible to set the same password again and change it at any time independent of PW Policy:
 
  "PasswordChangeOptions": {
    "UseAutomaticContext": false,
    "LdapHostname": "dcofother.domain.de",
    "LdapPort": 389,
    "LdapUsername": "svc_pwchange",
    "LdapPassword": "mypassword",
    "CheckRestrictedAdGroups": true,
...
Any ideas or hints?
